src/Components/PaperEditor/ProgrammeScript/index.js

```diff
@@ -523,211 +312,106 @@ class ProgrammeScript extends Component {
-                      {/* <FontAwesomeIcon icon={ faFileExport } />  */}
-                      ADL - Audio <FontAwesomeIcon icon={ faInfoCircle } />
-                    </Dropdown.Item>
-                    <Dropdown.Item
-                      onClick={ this.handleExportFCPX }
-                      title="export FCPX XML, to import the programme script as a sequence in Final Cut Pro X, video editing software"
-                    >
-                      FCPX <FontAwesomeIcon icon={ faInfoCircle } />
-                    </Dropdown.Item>
-                    <Dropdown.Divider />
-                    <Dropdown.Item
-                      onClick={ this.handleExportTxt }
-                      title="export Text, export the programme script as a text version"
-                    >
-                      Text File <FontAwesomeIcon icon={ faInfoCircle } />
-                    </Dropdown.Item>
-                    <Dropdown.Item
-                      onClick={ () => {
-                        alert('export word doc not implemented yet');
-                      } }
-                      title="export docx, export the programme script as a word document"
-                    >
-                      Word Document <FontAwesomeIcon icon={ faInfoCircle } />
-                    </Dropdown.Item>
-                    <Dropdown.Divider />
-                    <Dropdown.Item
-                      onClick={ this.handleExportJson }
-                      title="export Json, export the programme script as a json file"
-                    >
-                      Json <FontAwesomeIcon icon={ faInfoCircle } />
-                    </Dropdown.Item>
-                  </Dropdown.Menu>
-                </Dropdown>
-              </Col>
-              <Col sm={ 12 } md={ 1 }>
-                <Button
-                  variant="outline-secondary"
-                  onClick={ this.handleSaveProgrammeScript }
-                  // size="sm"
-                  title="save programme script"
-                  block
-                >
-                  <FontAwesomeIcon icon={ faSave } />
-                  {/* Save */}
-                </Button>
-              </Col>
-            </Row>
-          </Card.Header>
-
-          <Card.Body>
-            <article
-              style={ { height: '60vh', overflow: 'scroll' } }
-              onDoubleClick={ this.handleDoubleClickOnProgrammeScript }
-            >
-              {this.state.programmeScript ? (
-                <ProgrammeScriptContainer
-                  items={ this.state.programmeScript.elements }
-                  handleReorder={ this.handleReorder }
-                  handleDelete={ this.handleDelete }
-                  handleEdit={ this.handleEdit }
-                />
-              ) : null}
-            </article>
-          </Card.Body>
-        </Card>
-      </>
-    );
-  }
-}
+          </article>
+        </Card.Body>
+      </Card>
+    </>
+  );
+};
 
 ProgrammeScript.propTypes = {
```

Comment:
Have we lost the projectId and papereditId? We should set these for all of our props.

Reply:
The props are retrieved in the following way:
```js
  const papereditsId = props.match.params.papereditId;
  const projectId = props.match.params.projectId;
```
which means if you want to strictly verify for the two Ids, you'd need to make changes in line 406 :)